ament_cmake_nose/cmake/ament_add_nose_test.cmake

```diff
@@ -133,5 +133,3 @@ function(_ament_add_nose_test testname path)
     LABELS "nose"
   )
 endfunction()
-
-# TODO provide function to register all found tests separately
```

Comment:
Could such a function provide the ability to flag / label test functions independently (in the vein of what is suggested in https://github.com/ros2/build_cop/issues/41#issuecomment-317082214) ?